argocd/argocd/values.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 argo-cd:
-  installCRDs: false
+  installCRDs: true
```

Comment:
I can't remember why I set this to `false` but I'm pretty sure it's required.

Reply:
That was an attempt to fix the stack, as I'm currently getting errors due to the Application CRD not being deployed.

Reply:
Are you using master?

Reply:
Yes